bbb-common-web/src/main/java/org/bigbluebutton/presentation/imp/SvgImageCreatorImp.java

```diff
@@ -261,6 +264,13 @@ private boolean generateSvgImage(File imagePresentationDir, UploadedPresentation
                             height = imageResolution.getHeight();
                         }
 
+                        if(imageResolution.getWidth() > MAX_SVG_WIDTH || imageResolution.getHeight() > MAX_SVG_HEIGHT) {
+                            log.info("The image exceeds max dimension allowed, it will be resized.");
+                            imageResizer.resize(tempPng, MAX_SVG_WIDTH + "x" + MAX_SVG_HEIGHT);
+                            width = MAX_SVG_WIDTH;
+                            height = MAX_SVG_HEIGHT;

```

Comment:
```suggestion
                            imageResolution = imgResService.identifyImageResolution(tempPng);
                            width = imageResolution.getWidth();
                            height = imageResolution.getHeight();
```

`convert` will resize your image proportionally (preserving the aspect ratio), so the output will not have exactly the max allowed width and height. We need to check the new dimensions.